File: 하루에 한개씩 문제 풀기/Go/Programmers/Level 0/조건 문자열.go
func solution(ineq string, eq string, n int, m int) int {
	if ineq == "<" {
		if eq == "=" {
			if n <= m {
				return 1
			}
		} else if eq == "!" {
			if n < m {
				return 1
			}
		}
	} else if ineq == ">" {
		if eq == "=" {
			if n >= m {
				return 1
			}
		} else if eq == "!" {
			if n > m {
				return 1
			}
		}
	}
	return 0
}

File: 하루에 한개씩 문제 풀기/Java/Programmers/Level 0/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        if (ineq.equals("<")) {
            if (eq.equals("=")) {
                if (n <= m) {
                    return 1;
                }
            } else if (eq.equals("!")) {
                if (n < m) {
                    return 1;
                }
            }
        } else if (ineq.equals(">")) {
            if (eq.equals("=")) {
                if (n >= m) {
                    return 1;
                }
            } else if (eq.equals("!")) {
                if (n > m) {
                    return 1;
                }
            }
        }
        return 0;
    }
}
